### PR TITLE
Localize/minimize windows changes

### PR DESCRIPTION
**Pull request by stablebits**
- Minimize the number of modified files
- Reverted some changes that seem to be redundant (to be reverted again if needed)

### CODE REVIEW

File: include/quicly/loss.h [hunk 1]
```diff
@@ -226,14 +226,15 @@ inline uint32_t quicly_rtt_get_pto(quicly_rtt_t *rtt, uint32_t max_ack_delay, ui
...
 {
-    quicly_loss_t loss = {0};
-    loss.conf = conf;
-    loss.max_ack_delay = max_ack_delay;
-    loss.ack_delay_exponent = ack_delay_exponent;
-    loss.loss_time = INT64_MAX;
-    loss.alarm_at = INT64_MAX;
-    *r = loss;
-
+    *r = (quicly_loss_t){.conf = conf,
```
Comment (stablebits) [inline include/quicly/loss.h]: This code is c99 compatible and so it should be fine as is. If not, let's see the actual error.

**Review by dzianisv — APPROVED** [review]
